docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+	- string
+	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+	
+"gravity"  
+	- 3-vector
+	- Gravity to use in scene for physical modeling.
+
+"timestep" 
+	- double
+	- The timestep to use for forward simulation.
+ 
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+ 
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+ 
+"rigid object paths"
+	- list of strings
+	- A list of locations to query for supported object files that should be available to be loaded in the world.
+
+
+`Scene Attributes`_
+===================
+Scene Attributes templates hold relevant information describing a scene's mesh, geometry and physical properties.  These files should be named in the format : 
+
+ 	<scenename>.scene_config.json
+	
+Scene Mesh Handles And Types
+----------------------------	
+
+Below are the handles and descriptors for various mesh assets used by a scene.
+	
+Mesh types are included to support future format flexibility. In the JSON they are limited to string keys used by :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap`, which map to values in :ref:`AssetType`.  Any mesh type specification used in JSON but not found in :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap` results in that mesh type being set to :ref:`AssetType::UNKNOWN`.  JSON-Specified mesh types will override any default settings, such as those that might be derived from file name extensions.
+
+"render mesh" 
+	- string
+	- The name of the file describing the render mesh to be used by the scene. 
+"render mesh type"
+	- string
+	- The type of the render mesh. If not specified, value will be derived based on given **"render mesh"** file extension. 
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the scene. 
+"collision mesh type"
+	- string
+	- The type of the collision mesh. If not specified, value will be derived based on given **"collision mesh"** file extension.  Currently only "mp3d" is supported, other values default to :ref:`AssetType::UNKNOWN` 
+"semantic mesh"
+	- string
+	- The name of the file describing the scene's semantic mesh. 
+"semantic mesh type"
+	- string
+	- The type of the render mesh. If not specified, value will be derived based on given **"semantic mesh"** file extension.  Currently only "mp3d" is supported, other values default to :ref:`AssetType::INSTANCE_MESH`
+"house filename"
+	- string
+	- The name of the file containing semantic type maps and hierarchy. 
+"nav mesh"
+	- string
+	- The name of the file describing the NavMesh. 
+
+
+Scene Frame and Origin
+----------------------
+
+The tags below are used to build a coordinate frame for the scene, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.
```

Comment:
Is the user allowed to choose the handedness, or do we want to assert on a certain handedness here?

Beware, naively changing handedness can sometimes break things like backface-culling.

Reply:
Handedness seems to be defaulted in the system to be right-handed, and the esp::geo::CoordinateFrame that consumes these bases does not include a 3rd basis vector, so it doesn't seem that it can be changed.